### DISCUSSION TITLE
Box64 on AWS / Oracle Cloud

### DISCUSSION THREAD

**Discussion by couloirextreme** [ptitSeb/box64#186] [Ideas]
Has anyone tried Box64 on [AWS](https://aws.amazon.com/ec2/graviton/) or [Oracle Cloud](https://www.oracle.com/cloud/compute/arm/)? I've been playing with QEMU user-level emulation on the Oracle Cloud and have a performant [Minecraft Bedrock Server](https://www.minecraft.net/en-us/download/server/bedrock) running on Ubuntu, but would love to try this with Box64.

**Comment by suisseWalter** [comment]
I have run some factorio on oracle cloud (4core 24gb ram)vm. it works (just use the compile options for "other arm". but not sure how well miencraft bedrock performs with box64. I have noticed in my case that ressource utilisation is less then optimal. I ran benchmaks, which should put the cpu to 100% load at least on one core. but there where times where the cpu wasn't at 100% on one core. this normally doesn't happen with factorio benchmaks on x86-64. so don't expect stellar performance.

**Comment by couloirextreme** [comment]
I got it working on Oracle Cloud with just one issue which is documented [here](https://github.com/ptitSeb/box64/issues/175#issuecomment-997477289).
